source/includes/_cli.md

```diff
@@ -11,51 +11,52 @@ To generate files containing private and public keys `--new_account` flag should
 
 Example:
 ```bash
-iroha-cli --new_account --name alice@ru --pass_phrase mysupersecretpassword
+iroha-cli --new_account --account_name alice@ru --pass_phrase mysupersecretpassword --key_path ./
 ```
 
-After that `alice@ru.priv` and `alice@ru.pub` files will be generated in the folder where iroha-cli has been launched.
+After that `alice@ru.priv` and `alice@ru.pub` files will be generated in the `key_path` folder. By default `key_path` if not specified is the folder where iroha-cli has been launched.
 
 ## Interactive mode 
 
 Run: 
-<code class="bash"> iroha-cli --interactive --name your_account_id </code>
+<code class="bash"> iroha-cli --interactive --account_name your_account_id --key_path /path/to/keys </code>
 
-Your account_id will be used as the creator of the queries and transactions, signing up messages with accounts keys, fill up counters. 
-At the start of interactive cli will try to find locally keypairs associated with the account_id, hence this keypairs must be generated and stored locally. 
+Your `account_id` is used as the creator of the queries and transactions for signing up messages with accounts keys and fill up counters. 
+You can specify the location of the keypairs associated with the `account_id` using `key_path` flag. By default, current folder is used.
 
 ## Starting menu 
 
 At the start of cli user has two possible modes:
 
 * Start a transaction
 * Start a query
+* Get status of a transaction
 
 ![image](../images/cli/image.png)
 
 ## Transaction CLI
 
 Start up of transaction cli will trigger the creation of a new transaction.
-Each transaction consists of commands, being less or equal than **65000**. User is offered to add commands to new formed transaction.
-All meta data of transaction will be filled automatically (signature, tx_counter, creator account, timestamp).
-Currently following Iroha commands are supported in Interactive CLI:
+Each transaction consists of commands, being less or equal than **65000**. User is offered to add commands to anew transaction.
```

Comment:
equal than? Better to say being equal or less than ...

Reply:
to **anew**

Reply:
anew is a legit word